gulpfile.js

```diff
@@ -6,7 +6,7 @@ var fs = require('fs');
 
 var mappings = {
     'batchService': {
-        'source': 'batch/2017-05-01.5.0/swagger/BatchService.json',
+        'source': 'specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json',
```

Comment:
This repo is only for azure batch sdk. So I think this is fine to just reference to batch swagger spec.